go.mod

```diff
@@ -158,17 +158,17 @@ require (
 	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/syslogexporter v0.96.0 // indirect
 	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/zipkinexporter v0.96.0 // indirect
 	github.com/open-telemetry/opentelemetry-collector-contrib/extension/storage v0.96.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/internal/common v0.96.0 // indirect
-	github.com/open-telemetry/opentelemetry-collector-contrib/internal/coreinternal v0.96.0 // indirect
-	github.com/open-telemetry/opentelemetry-collector-contrib/internal/kafka v0.96.0 // indirect
+	github.com/open-telemetry/opentelemetry-collector-contrib/internal/common v0.97.0 // indirect
+	github.com/open-telemetry/opentelemetry-collector-contrib/internal/coreinternal v0.97.0 // indirect
+	github.com/open-telemetry/opentelemetry-collector-contrib/internal/kafka v0.97.0 // indirect
 	github.com/open-telemetry/opentelemetry-collector-contrib/internal/sharedcomponent v0.96.0 // indirect
```

Comment:
we don't upgrade piecemeal, it should include all OTEL packages

Reply:
@yurishkuro think I have update all OTEL packages, ready for another review 

PS: I saw this warning in my IDE
![image](https://github.com/jaegertracing/jaeger/assets/6630779/8b16669d-5a6a-4492-8465-26d125b1fe35)